Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,25 +1,78 @@
-function setAlarm() {}
+var audio = new Audio("alarmsound.mp3");
 
-// DO NOT EDIT BELOW HERE
+function setAlarm() {
+  const time = parseInt(document.getElementById("alarmSet").value); // Fetch the value of the input in seconds
 
-var audio = new Audio("alarmsound.mp3");
+  document.querySelector(".quadrat").classList.add("disabled"); // Disable the .quadrat element
 
-function setup() {
-  document.getElementById("set").addEventListener("click", () => {
-    setAlarm();
-  });
+  if (isNaN(time) || time <= 0) {
+    alert("Please enter a valid number of seconds.");
+    return;
+  }
 
-  document.getElementById("stop").addEventListener("click", () => {
-    pauseAlarm();
-  });
+  // Display the initial time remaining
+  document.getElementById("timeRemaining").innerHTML =
+    "Time Remaining: " + formatTime(time);
```

Comment:
You can also place these code inside `startCountdown()` so that all the code to update the clock display is in once place.